addons/account_peppol/models/account_move.py

```diff
@@ -70,3 +71,18 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = False
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False, force_email_company=False, force_email_lang=False):
+        render_context = super()._notify_by_email_prepare_rendering_context(
+            message, msg_vals, model_description=model_description,
+            force_email_company=force_email_company, force_email_lang=force_email_lang
+        )
+        if model_description == 'Invoice':
+            invoice = render_context['record']
+            invoice_country = invoice.commercial_partner_id.country_code
+            if invoice_country in PEPPOL_DEFAULT_COUNTRIES:
```

Comment:
@smetl I changed it even a bit more further in the direction of your suggestion.
Then I don't need to know if "is_peppol_country", the info will just not be there if its not.